azurerm/resource_arm_virtual_network_gateway_connection_test.go

```diff
@@ -341,3 +360,89 @@ resource "azurerm_virtual_network_gateway_connection" "test_2" {
 }
 `, rInt, rInt2, sharedKey, location, altLocation)
 }
+
+func testAccAzureRMVirtualNetworkGatewayConnection_ipsecpolicy(rInt int, location string) string {
+	return fmt.Sprintf(`
+variable "random" {
+  default = "%d"
+}
+
+resource "azurerm_resource_group" "test" {
+  name = "acctestRG-${var.random}"
+  location = "%s"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name = "test-${var.random}"
```

Comment:
can we make this `acctestvn` to be picked up by the acceptance testing sweepers?

Reply:
Sure, this functionality was new to me.